index.bs

```diff
@@ -1322,13 +1322,13 @@ sends only the result to the authenticator. The authenticator signs over the com
 
 The goals of this design can be summarized as follows.
 
-- The scheme for generating signatures should accommodate cases where the link between the client platform and authenticator
+- The scheme for generating signatures SHOULD accommodate cases where the link between the client platform and authenticator
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done.